src/components/Connect/Ledger/index.test.js

```diff
@@ -3,17 +3,19 @@ import { initialState as wpInitialState } from '../../../services/WalletProvider
 import { initialLedgerState } from '../../../services/WalletProvider/ledgerUtils'
 import composeMockAppTree from '../../../test-utils/composeMockAppTree'
 import { mockFetchDefaultWallet } from '../../../test-utils/composeMockAppTree/createWalletProviderContextFuncs'
-import ConnectLedger from '.'
+import { ConnectLedger } from '.'
 import { flushPromises } from '../../../test-utils'
 import { TESTNET_PATH_CODE } from '../../../constants'
 import createPath from '../../../utils/createPath'
 
 describe('Ledger configuration', () => {
   let backSpy, nextSpy
-  afterEach(() => {
-    jest.clearAllMocks()
+  beforeEach(() => {
```

Comment:
`backSpy` and `nextSpy` were initially passed as `undefined` because they were set in `afterEach`